tracim/tracim/tests/library/test_workspace.py

```diff
@@ -21,3 +27,17 @@ def test_children(self):
         workspace = DBSession.query(Workspace).filter(Workspace.label == 'workspace_1').one()
         folder = ContentApi.get_canonical_query().filter(Content.label == 'folder_1').one()
         eq_([folder, ], list(workspace.get_valid_children()))
+
+    def test_get_notifiable_roles(self):
```

Comment:
Generally, its a good reflex to make two tests for testing nominal case ok and nominal case error.

In the current case, this would be named like this: 

```
def test_get_notifiable_roles__ok__nominal_case(self): 
    [...]

def test_get_notifiable_roles__err__nominal_case(self): 
    [...]
```

By the way we have test naming conventions. See http://algoo.trac.im/workspaces/11/folders/1194/pages/2043 section "Tests et nommage des méthodes de test"

This is not for fix now but for information (for future developments).

You can merge the PR